jujugui/static/gui/src/app/components/deployment-flow/credential/add/test-add.js

```diff
@@ -680,7 +681,8 @@ describe('DeploymentCredentialAdd', function() {
           close={sinon.stub()}
           cloud={{id: 'google', name: 'google'}}
           clouds={clouds}
-          regions={[{name: 'test-region'}]}
+          generateCloudCredentialTag={sinon.stub()}
+          regions={[{name: 'test-regiZon'}]}
```

Comment:
I'm not sure it can even get to this point if the info is missing - I''ll investigate further.